engine/src/flutter/flow/layers/performance_overlay_layer_unittests.cc

```diff
@@ -151,26 +158,25 @@ class ImageSizeTextBlobInspector : public virtual DlOpReceiver,
     text_positions_.push_back(DlPoint(x, y));
   }
 
-  const std::vector<DlISize>& sizes() { return sizes_; }
+  const std::vector<DlSize>& sizes() { return sizes_; }
   const std::vector<std::shared_ptr<DlText>> texts() { return texts_; }
   const std::vector<DlPoint> text_positions() { return text_positions_; }
 
  private:
-  std::vector<DlISize> sizes_;
+  std::vector<DlSize> sizes_;
   std::vector<std::shared_ptr<DlText>> texts_;
   std::vector<DlPoint> text_positions_;
 };
 
-TEST_F(PerformanceOverlayLayerTest, PaintingEmptyLayerDies) {
+TEST_F(PerformanceOverlayLayerTest, PaintingEmptyLayerDoesNotDie) {
   const uint64_t overlay_opts = kVisualizeRasterizerStatistics;
   auto layer = std::make_shared<PerformanceOverlayLayer>(overlay_opts);
 
   layer->Preroll(preroll_context());
   EXPECT_EQ(layer->paint_bounds(), DlRect());
   EXPECT_FALSE(layer->needs_painting(paint_context()));
 
-  // Crashes reading a nullptr.
-  EXPECT_DEATH_IF_SUPPORTED(layer->Paint(paint_context()), "");
+  layer->Paint(paint_context());
```

Comment:
Since the test is "does not die" it's worth commenting that this is where we are checking this doesn't die.  I wish this method returned an error we could assert on.